[ask]
`npm run start`した際に以下のslick-carouselのモジュールがない旨エラーが発生しているのですが、ご確認いただけますか…！（package.jsonにライブラリ追加した差分PRに含められてます？）


```
ERROR in ./src/js/app.js 2:0-24
Module not found: Error: Can't resolve 'slick-carousel' in '/Users/sato/Development/gizumo/front-coding/src/js'
resolve 'slick-carousel' in '/Users/sato/Development/gizumo/front-coding/src/js'
  Parsed request is a module
  using description file: /Users/sato/Development/gizumo/front-coding/package.json (relative path: ./src/js)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /Users/sato/Development/gizumo/front-coding/src/js/node_modules doesn't exist or is not a directory
      /Users/sato/Development/gizumo/front-coding/src/node_modules doesn't exist or is not a directory
      looking for modules in /Users/sato/Development/gizumo/front-coding/node_modules
        single file module
          using description file: /Users/sato/Development/gizumo/front-coding/package.json (relative path: ./node_modules/slick-carousel)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/sato/Development/gizumo/front-coding/node_modules/slick-carousel doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/sato/Development/gizumo/front-coding/node_modules/slick-carousel.js doesn't exist
        /Users/sato/Development/gizumo/front-coding/node_modules/slick-carousel doesn't exist
      /Users/sato/Development/gizumo/node_modules doesn't exist or is not a directory
      /Users/sato/Development/node_modules doesn't exist or is not a directory
      /Users/sato/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory

webpack 5.95.0 compiled with 1 error in 2335 ms
```

--- a/package.json
+++ b/package.json
@@ -34,7 +34,8 @@
     "webpack-dev-server": "^4.9.2"
   },
   "dependencies": {
-    "jquery": "^3.6.1"
+    "jquery": "^3.6.1",
+    "slick-carousel": "^1.8.1"
   },
   "browserslist": [
     "ie >= 11",

--- a/src/css/layout/header.scss
+++ b/src/css/layout/header.scss
@@ -157,25 +157,25 @@
   }
 
   &__hamburger.active {
-    span:first-of-type{
+    span:first-of-type {
       top: 50%;
       transform: rotate(45deg);
     }
   }
   
   &__hamburger.active {
-    span:nth-of-type(2){
+    span:nth-of-type(2) {
       opacity: 0;
-   }
+    }
   }
   &__hamburger.active {
-    span:last-of-type{
+    span:last-of-type {
       top: 50%;
       transform: rotate(-45deg);
     }
   }
-  &__slide-menu.active{
-    transform: translateX(0); 
+  &__slide-menu.active {
+    transform: translateX(0);
   }
   .body {
     height: 100%;

--- a/src/js/app.js
+++ b/src/js/app.js
@@ -1,5 +1,5 @@
 import $ from "jquery";
-import 'slick-carousel'
+import "slick-carousel"
 
 
 $(function() {

